.github/workflows/haskell-ci.yml

```diff
@@ -30,6 +30,8 @@ jobs:
     strategy:
       matrix:
         include:
+          - ghc: 9.2.1
+            allow-failure: false
```

Comment:
Let's see whether that's sufficient for making CI work with GHC 9.2.1.

Usually you'd update the `tested-with` field in `newtype-generics.cabal` and then run `haskell-ci regenerate`.